File: app/src/main/java/can/siempredelao/f1kotlin/backend/FastestLap.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class FastestLap {

    private String       rank;
    private String       lap;
    @SerializedName("Time")
    private Time_        time;
    @SerializedName("AverageSpeed")
    private AverageSpeed averageSpeed;

}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/FastestLap.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

class FastestLap(val rank: String,
                 val lap: String,
                 @SerializedName("Time") val time: Time_,
                 @SerializedName("AverageSpeed") val averageSpeed: AverageSpeed)
